src/server/game/Globals/ObjectMgr.h

```diff
@@ -1013,6 +1013,7 @@ class ObjectMgr
         void LoadPetNumber();
         void LoadCorpses();
         void LoadFishingBaseSkillLevel();
+        void ChangeFishingBaseSkillLevel(uint32 entry, int32 skill);
```

Comment:
where is this method used?

Reply:
I use this in module for some scripts

Reply:
It's just that when I needed to change the value in the map, I realized that I couldn't change it, because the map is private.